crud/homework2.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE agents
+SET agents.COMISSION = agents.COMISSION - 0.02
+WHERE agents.AGENT_CODE = 
+(
+    SELECT customer.AGENT_CODE
+    FROM customer
+    GROUP BY customer.AGENT_CODE
+    ORDER BY SUM(customer.PAYMENT_AMT)
+    LIMIT 1
+)
+;
```

Comment:
Because there is not only one agent who have min payment amount, so the SQL should be like

```SQL
UPDATE agents
SET commission=commission - .02
WHERE agent_code IN
(
SELECT agent_code FROM customer
WHERE PAYMENT_AMT=(SELECT MIN(PAYMENT_AMT) FROM customer)
);
```